features/step_definitions/networking.rb

```diff
@@ -790,6 +790,25 @@
   logger.info "Subnet range for the primary interface on the node is stored in the #{cb_name} clipboard."
 end
 
+Given /^the node's MTU value is stored in the#{OPT_SYM} clipboard$/ do |cb_node_mtu|
+  ensure_admin_tagged
+  cb_node_mtu = "mtu" unless cb_node_mtu
+  @result = admin.cli_exec(:get, resource: "network.operator", output: "jsonpath={.items[*].spec.defaultNetwork.type}")
```

Comment:
how about using `oc debug node/xxx chroot /host ip route show default` to get the mtu, then we do not need to check the networkType.

Reply:
Ya this was discussed earlier in some cases. Good approach is to get required pmtrs from network pods instead of node as oc debug node tends to throw intermittent stderr issues may be due to auto hiccups causing premature failures. WDYT